storybook/cypress/e2e/autosuggest.cy.ts

```diff
@@ -414,4 +414,19 @@ describe("Autosuggest", () => {
       .get("mark")
       .should("contain", "omg");
   });
+
+  it("should limit the block-size of the listbox to 10 items or available block-size within viewport", () => {
+    cy.get("@input").focus().type("sugg");
+    cy.wait(200);
+    cy.get(".listbox-container").should("have.attr", "style", "max-block-size: 338px;");
+    cy.matchImageSnapshot();
+
+    cy.get("@input").focus().clear();
+    cy.viewport(1000, 338);
+    cy.wait(200);
+    cy.get("@input").focus().type("sugg");
+    cy.wait(200);
+    cy.get(".listbox-container").should("have.attr", "style", "max-block-size: 237px;");
```

Comment:
Nu test je of er styling is gezet. Terwijl je eigenlijk wilt testen wat de maximale hoogte is. Daar zit een verschil in.

Je kan tegen de computed styling aantesten met `.should('have.css')`: https://docs.cypress.io/guides/references/assertions#CSS

Reply:
Ik heb deze test gesplitst. En neem van beide een matchImageSnapshot(). Vind je dat voldoende?

Reply:
Ik denk het wel :+1: